src/System.ComponentModel.TypeConverter/ref/project.json

```diff
@@ -1,12 +1,28 @@
 {
   "dependencies": {
     "System.Collections.NonGeneric": "4.3.0-beta-24522-03",
+    "System.Collections.Specialized": "4.3.0-beta-24522-03",
```

Comment:
Please trim this to the exact set that is needed. I really hope we don't need things like System.Security.Permissions for example.